docs/installation/maven-install.md

```diff
@@ -8,128 +8,73 @@ The Legend platform has five components. The minimum required to run Legend are
 
 - **Engine**: Provides a Pure parser and compiler that executes plans when provided with a Pure function, a mapping, and a runtime. It's also an access point for model transformers written using the Legend language.
 - **SDLC**: Provides a rich REST API letting users manage metadata. Most SDLCs are file and text-centric, but the Legend SDLC is model-centric, meaning users interact with model entities rather than with files and folders. The Legend SDLC enables:
-    - Users to develop with tools designed for editing models (rather than files or code).
-    - Users to view changes with tools designed for viewing model-level changes (rather than text changes).
-    - Clients to create their own tools for their own use cases.
-- **Studio**: Lets you describe and connect data in business terms to create data models.
-- **Pure**: Lets you alter Pure, the underlying language for Legend.
+  - Users to develop with tools designed for editing models (rather than files or code).
+  - Users to view changes with tools designed for viewing model-level changes (rather than text changes).
+  - Clients to create their own tools for their own use cases.
+- **Studio**: Provides a way to describe and connect data in business terms to create data models.
+- **Pure**: Provides a way to alter Pure, the underlying language for Legend.
 - **Shared**: Provides shared code used across Legend apps for universal server-side functionality, like hosting static files, performing authentication, and so on.
 
 ## Prerequisites
 
 - A GitLab.com account or your own GitLab server.
-  - Create an *Application* by going to **profile settings > applications**.
+
+  Create an _Application_ by navigating to `User Settings > Applications` and create an application with the following parameters:
+
+  - Name: Legend Demo
+  - Enable the "Confidential" check box
+  - Enable these scopes: openid, profile, api
+  - Redirect URI: Add the followings:
+
+```sh
+http://localhost:7070/api/auth/callback
+http://localhost:7070/api/pac4j/login/callback
+```
+
 - Java Development Kit (JDK) 8 or later.
```

Comment:
Each project's development setup instructions mention JDK 11. We should change to be consistent.
(We build with JDK11 but target JDK8).

Reply:
Updated to 11 !